problems/arrayMethods.js

```diff
@@ -119,7 +196,19 @@ const tripledAndLessThan20 = () => {};
  * @returns {number[]} Numbers evenly divided by divisor.
  */
 
-const divisibleBy = () => {};
+const divisibleBy = (divisor, nums) => {
+  /* let filterArr = [];
+  let output = nums.map((el) => {
+    return (el % divisor);
+  })
+  
+  return filterArr = output.filter((el) => {
```

Comment:
Try writing this function with only .filter() instead of doing .map() first.

Reply:
I think too much and I am making the problems more complicated. Thanks.